app/build.gradle

```diff
@@ -57,7 +57,7 @@ android {
 
 	dexOptions {
 		preDexLibraries !project.hasProperty('buildServer')
-		javaMaxHeapSize "1g"
+		javaMaxHeapSize "2g"
```

Comment:
Maybe try to remove those dexOptions completely. They might not be needed anymore. Also, the max heap size here might not affect the lint process. That's gradle options.